main/pipeline/pipeline.h

```diff
@@ -40,7 +40,8 @@ typecheck(std::unique_ptr<core::GlobalState> &gs, std::vector<ast::ParsedFile> w
           WorkerPool &workers, bool cancelable = false,
           std::optional<std::shared_ptr<core::lsp::PreemptionTaskManager>> preemptionManager = std::nullopt);
 
-ast::ParsedFile typecheckOne(core::Context ctx, ast::ParsedFile resolved, const options::Options &opts);
+ast::ParsedFile typecheckOne(core::Context ctx, ast::ParsedFile resolved, const options::Options &opts,
```

Comment:
`ctx.state` is actually a `core::GlobalState &`, so you don't need to pass another argument. :)

Reply:
Good to know, but this is a change I forgot to remove when I moved the error queue stuff into `pipeline::typecheck`